### PR TITLE
[CUDOS-720] Added basic automated tests

### PR DESCRIPTION
**Pull request by avalkov**

### CODE REVIEW

File: integration_tests/test.sh [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+if [ "$CONTRACT" == "" ];then
+    printf '%s\n' "Failed to init the smart contract on the chain. Check the tx result above for a detailed error info" >&2 # write error message to stderr
+    printf '%s\n' "Failed INIT Transaction: ""$RES"" " >&2 # write error message to stderr
+
+    exit 1
+fi
+
+# ISSUE DENOM
+echo "ISSUE DENOM"
+ISSUE_DENOM_MSG=$(read_json './cosmwasm-testing/msgs/issue_denom_msg.json')
```
Comment (Infra1515) [inline integration_tests/test.sh]: When trying to execute it -  No such file or directory. I don't see a cosmwasm-testing folder, perhaps you meant integration_tests?

**Review by Infra1515 — CHANGES_REQUESTED** [review]
Need to add a readme - how exactly to use it.
If you just start a node in the background(in a different directory then where the test is located) and run the integration tests - it will error out and say 
```Error: faucet.info: key not found```
Could you please add a readme - what is this file - purpose, how to use it ( bullet point instructions ) and so on.
Currently it does not work

File: integration_tests/test.sh (hunk 1 above)
Comment (Infra1515) [reply integration_tests/test.sh]: Also on the same line - when I fix the paths (($pwd)/msgs) - I get ```Error: rpc error: code = InvalidArgument desc = failed to execute message; message index: 0: dispatch: submessages: denom symbol() can not be space: invalid denom symbol: invalid request```
Please check it

Comment (avalkov) [reply integration_tests/test.sh]: Improved how the testa are ran, check the README.md

**Pull request merged**